io.github.endless_sky.endless_sky.appdata.xml

```diff
@@ -22,28 +22,40 @@
   <url type="homepage">https://endless-sky.github.io/</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://user-images.githubusercontent.com/20871346/118372384-fd937580-b576-11eb-889f-c51dc95eda61.png</image>
-      <caption>The game's "new pilot" scenario-picking screen</caption>
+      <image>https://cdn.cloudflare.steamstatic.com/steam/apps/404410/ss_af75baf860ad0dddcdc86f00dfb7b3ff3761133f.jpg</image>
+      <caption>The three starter ships flying in formation over a Republic planet.</caption>
     </screenshot>
     <screenshot>
-      <image>https://endless-sky.github.io/images/screenshots/battle.jpg</image>
-      <caption>A chaotic battle between the Republic and the Free Worlds</caption>
+      <image>https://cdn.cloudflare.steamstatic.com/steam/apps/404410/ss_c7f18a682e8df27368c2c6a421624a395719c6c5.jpg</image>
+      <caption>A battle between Free World ships and pirates.</caption>
     </screenshot>
     <screenshot>
-      <image>https://endless-sky.github.io/screenshots/beams.jpg</image>
-      <caption>Diverse weaponry abounds in the Endless Sky universe</caption>
+      <image>https://cdn.cloudflare.steamstatic.com/steam/apps/404410/ss_93d13da0bdfe2c4b987eb4f5e06fb67344df518d.jpg</image>
+      <caption>The shipyard view, used to buy and sell different kinds of ships.</caption>
     </screenshot>
     <screenshot>
-      <image>https://camo.githubusercontent.com/647979e73d2121a1a982a96b0561e0e435598fd4/68747470733a2f2f656e646c6573732d736b792e6769746875622e696f2f73637265656e73686f74732f736d616c6c382e6a7067</image>
-      <caption>Exotic space creatures</caption>
+      <image>https://cdn.cloudflare.steamstatic.com/steam/apps/404410/ss_df5a65876c0a23d178f047d53ed5f9944a6c438b.jpg</image>
+      <caption>The outfitter view, used to outfit a ship with different weapons, engines, and other systems.</caption>
     </screenshot>
     <screenshot>
-      <image>https://endless-sky.github.io/images/screenshots/outfitter.jpg</image>
-      <caption>Customize items installed on your ship(s) at a planet's Outfitter</caption>
+      <image>https://cdn.cloudflare.steamstatic.com/steam/apps/404410/ss_4467e08dadf09d4bbaecf713e56faf134e590980.jpg</image>
+      <caption>View of a conversation with a local Captain.</caption>
```

Comment:
```suggestion
      <caption>View of a conversation with a local captain.</caption>
```
I don't think captain should be capitalised here, since it's just being used as a word and not specifically someone's title. But, I would like another opinion.

Reply:
It's capitalized in the game when referring to a specific captain, so I thought I would do it here too. Not sure though yeah